pkg/loadbalancer/redirectpolicy/main_test.go

```diff
@@ -10,5 +10,15 @@ import (
 )
 
 func TestMain(m *testing.M) {
-	goleak.VerifyTestMain(m)
+	goleak.VerifyTestMain(m,
+		// The metrics "status" collector tries to connect to the agent and leaves these
+		// around. We should refactor pkg/metrics to split it into "plain registry"
+		// and the agent specifics.
+		goleak.IgnoreTopFunction("net/http.(*persistConn).writeLoop"),
+		goleak.IgnoreTopFunction("internal/poll.runtime_pollWait"),
+
+		// Unfortunately we don't have a way for waiting for the workqueue's background goroutine
+		// to exit (used by pkg/k8s/resource), so we'll just need to ignore it.
+		goleak.IgnoreTopFunction("k8s.io/client-go/util/workqueue.(*Typed[...]).updateUnfinishedWorkLoop"),
```

Comment:
This exception exists in a few places in the codebase, is it possible to make it global?

Reply:
I'll follow up with this in another PR since we've got bunch of places that also use the wrong ignore.